### ISSUE TITLE
->getFrames() returns null on loaded gifs

### ISSUE THREAD

**Issue by beryllium** [imanee/imanee#59]
After loading a gif from disk, I was expecting ->getFrames() to return an array of Imanee objects, but I got "null" back.

It seems that Imanee doesn't support this method on gifs loaded from disk, only ones assembled in memory. I was hoping to use Imanee to do some post-processing on existing gifs, so this is a bit of a blocker for that.

**Comment by beryllium** [comment]
FWIW I'm looking into the dark heart of Imagick to see if I can solve this & push some improvements back to Imanee.

**Comment by erikaheidi** [comment]
Yes, indeed, currently the getFrames() is only working for gifs created with Imanee. This would be an awesome feature, I will have a look here too and see what could be done. Thanks for bringing this up!

**Comment by strayobject** [comment]
Is anyone working on this? If not I would like to take a shot at it over the weekend (Monday at the latest).

**Comment by beryllium** [comment]
I had been looking into it but I wasn't able to resolve it to my satisfaction. You're welcome to take a shot, it would be much appreciated :)

**Comment by strayobject** [comment]
Cool, I already have a working code, just need to add couple checks and write tests for it :)

**Comment by strayobject** [comment]
Created a pull request, @beryllium if you have some time to test it before @erikaheidi reviews, please let me know if this is what you were expecting. (https://github.com/strayobject/imanee)
